api/queries_projects_v2.go

```diff
@@ -61,11 +61,27 @@ func UpdateProjectV2Items(client *Client, repo ghrepo.Interface, addProjectItems
 
 // ProjectsV2ItemsForIssue fetches all ProjectItems for an issue.
 func ProjectsV2ItemsForIssue(client *Client, repo ghrepo.Interface, issue *Issue) error {
+	type projectV2ItemStatus struct {
+		StatusFragment struct {
+			OptionID string `json:"optionId"`
+			Name     string `json:"name"`
+		} `graphql:"... on ProjectV2ItemFieldSingleSelectValue"`
+	}
+
+	type projectV2Item struct {
+		ID      string `json:"id"`
+		Project struct {
```

Comment:
Do we want to change this to be `ProjectV2ItemProject` now?

Reply:
It doesn't seem to matter since it's private and we never construct it.

Reply:
Does it make the loop below through `query.Repository.Issue.ProjectItems.Nodes` slightly easier though? Or rather less code.

Reply:
Mmm, I wouldn't do it for less code but maybe because then we'd know if one changed the other should too. I dunno, it seems tenuous. There's also a nice advantage to treating these things separately since if we decide we need more shurcool magic in one we can avoid it in the other.

I'd favour treating them separately and adding a comment later (not blocking this PR cause I don't have time right now)

Reply:
> There's also a nice advantage to treating these things separately since if we decide we need more shurcool magic in one we can avoid it in the other.

This has been one of the struggles I've experienced using shurcool library, whether the query structure is what I pass back to code or having separate structures that don't couple logic to graphql.